Add some additional categories to the desktop file

dGr8LookinSparky opened the pull request.

dGr8LookinSparky commented:
I'm still very new to flatpak, but some people indicated to me that Tremulous wasn't fully categorized well, does the included commit commit remedy that, or doe one or more additional files need categories specified as well?

AsciiWolf commented:
Thanks for your PR, however I am not sure if I really want these non-standard categories to be in the desktop file. I checked desktop files from various Linux distributions that have Tremulous packaged and none of them had such categories in them.

By the way, let me know if you (or other GrangerHub devs) want push access to this repo. :-)

dGr8LookinSparky commented:
Cool, I would appreciate having that push access, thanks :) !  Also thank you for setting up and maintaining this flatpak support for Trem!  I only recently found out about this flatpak support when taith made a post about it on the GHub forums (https://forum.grangerhub.com/t/lets-spam-reviews-on-linux-mint-community/4948/1), that's where he also mentioned that it wasn't fully categorized.  Although I wonder if he was looking somewhere else that that uses something other than the desktop categories, since he mentioned it wasn't listed under "Games" but you do have "Game" specified in the desktop file, I'm not sure if the lack of `s` plays a factor.  Like is there a way of specifying categories in the appdata file?

Is there a list of standard desktop file categories I could look at?  Perhaps there are some other categories that may apply to trem as well.  What ever we can do to have Tremulous show up appropriately in various related searches more often the better, even if previous distributions of Trem may have overlooked such categorization.

AsciiWolf commented:
@barthalion Feel free to give @dGr8LookinSparky a push rights to this repo. :-)

AsciiWolf commented:
Yeah, and maybe `Shooter` as well. Anyway, I probably know what taith was talking about. Flatpak apps in Linux Mint's Software Center are placed in special "Flatpak" category instead of the regular categories. This is a feature and we can't do anything with it. :-( They are however in regular categories on other distros (Ubuntu, Fedora etc.).

nedrichards commented:
@dGr8LookinSparky @AsciiWolf invitation to the repo sent.

dGr8LookinSparky commented:
I removed all of the non-standard categories from the desktop file except X-Shooter, and added the standard category StrategyGame in this commit: https://github.com/flathub/com.grangerhub.Tremulous/pull/2/commits/89776b9660eef664cdc7dec29f592150f6eae873

AsciiWolf commented:
Shooter is also a standard category, so it should be without the `X-` prefix. :-)

dGr8LookinSparky commented:
Sorry about that, I fixed it just now in this commit: https://github.com/flathub/com.grangerhub.Tremulous/pull/2/commits/dc95e66bb0f5394801bf857938c20b7bfa6a9653
